core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -1,21 +1,36 @@
 <script type="text/ng-template" id="components/socialButtons">
+<style type="text/css">
+  
+i.oppia-share-btn-fa {
+    display: inline-block;
```

Comment:
@shaz13 I think you're commenting in the wrong place? The comment you are replying to was made by me, not by @markhalpin.